src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped its work");

```

Comment:
The stop message should also be more informative. It's recommended to rephrase it to something like 'Excavator has finished working.' to clearly indicate the status of the machine.